adblock-lean

```diff
@@ -26,6 +26,16 @@ adblock-lean custom commands:
 	gen_config	generate default config
 	update		update adblock-lean to the latest version"
 
+mkdir -p /var/run/adblock-lean
+
+trap cleanup INT TERM EXIT
```

Comment:
I think you can also include QUIT and HUP. Don't ask me what each does :) People just do that :)